frontend/packages/dev-console/src/components/import/ImportForm.tsx

```diff
@@ -85,6 +85,20 @@ const ImportForm: React.FC<ImportFormProps & StateProps> = ({
       replicas: 1,
     },
     labels: {},
+    limits: {
+      cpu: {
+        request: 1,
```

Comment:
These are tiny initial values. It'd be better to the fields blank and make them optional.